deliciousbrains/wp-alert-bars
# Allow user to specify additional class to be added to alert bar

resolves #16 

**Steps to test**

- Clone repo.
- Update local dbi site composer.json to pull deliciousbrains/wp-alert-bars from the local repo.
-  From the admin panel create a new alert bar.
- You should see a 'Additional Alert Bar Class' custom field. Add a class name to this field and save,
- Verify in the site front end that this class is being added to the alert bar.
